plugins/modules/vmware_content_deploy_ovf_template.py

```diff
@@ -50,22 +59,29 @@
       description:
       - Name of the datastore to store deployed VM and disk.
       type: str
-      required: True
+      required: False
+    datastore_cluster:
+      description:
+      - Name of the datastore cluster housing a datastore to store deployed VM and disk.
+      - If datastore is not specified, the recommended datastore from this cluster will be used.
+      type: str
+      required: False
+      version_added: '1.9.0'
     folder:
       description:
       - Name of the folder in datacenter in which to place deployed VM.
       type: str
-      required: True
+      default: 'vm'
```

Comment:
We are changing the default and required value. Please add a changelog entry for this change.

Reply:
There are several of these that I am changing. Besides the default folder, I am changing the default storage_provisioning to 'thin', as this is best practice with today's flash storage. Also, previously required fields, like 'host' and 'datastore', will no longer be required as 'cluster' and 'datastore_cluster' can be used in their stead. I am working on a changelog to cover all aspects of this PR.